plugins/rapid7_insightidr/komand_rapid7_insightidr/actions/set_disposition_of_investigation/schema.py

```diff
@@ -66,27 +66,18 @@ class SetDispositionOfInvestigationOutput(insightconnect_plugin_runtime.Output):
     }
   },
   "required": [
-    "investigation"
+    "created_time",
+    "disposition",
+    "investigation",
+    "last_accessed",
+    "organization_id",
+    "priority",
+    "rrn",
+    "source",
+    "status",
+    "title"
```

Comment:
As investigation is the only property here, having the extra that are added from the refresh plugin will cause issues 
```
"required": [
    "investigation"
]
```